pkgs/tools/audio/beets/default.nix

```diff
@@ -46,13 +57,20 @@ lib.makeExtensible (self: {
   beets-minimal = self.beets.override { disableAllPlugins = true; };
 
   beets-unstable = callPackage ./common.nix {
-    inherit python3Packages extraPatches;
-    version = "2.2.0-unstable-2024-12-02";
+    inherit python3Packages;
+    extraPatches = [ ];
```

Comment:
This is the default right? If it is, then it can be removed?

Reply:
oh, right, duh. thanks!